src/containers/PageContainer/elements/Header/elements/InfoButton/elements/InfoPanel/InfoPanel.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { ConnectWalletButton } from '../../../../elements';
+import { InfoButtonProps } from '../../InfoButton';
+import { useInfoPanelHandlers } from './hooks';
+import {
+	WILDER_WORLD_OPTIONS,
+	ZERO_TECH_OPTIONS,
+	ZNS_OTHER_OPTIONS,
+} from './InfoPanel.constants';
+import './_info-panel.scss';
+
+interface InfoPanelProps extends InfoButtonProps {
+	onClose: () => void;
+}
+
+export const InfoPanel: React.FC<InfoPanelProps> = ({
+	isDesktop,
+	onClose,
+	onConnectWallet,
+}) => {
+	const handlers = useInfoPanelHandlers({
+		props: {
+			onClose,
+			onConnectWallet,
+		},
+	});
+
+	return (
+		<div className="info-panel__container border-primary border-rounded blur">
+			<div className="info-panel__content-section connect-wallet">
+				<ConnectWalletButton
+					onConnectWallet={handlers.handleConnectWallet}
+					isDesktop={isDesktop}
+				/>
+			</div>
+			<div className="info-panel__content-section">
+				<div className="info-panel__content-section-title">
+					<h3> Wilder World</h3>
+					<span className="divider"></span>
+				</div>
+				<div className="info-panel__content-section-body">
+					{Object.values(WILDER_WORLD_OPTIONS).map((option) => (
+						<div className="info-panel__content-section-body-item max-4">
+							<a
+								target="_blank"
+								rel="noreferrer"
+								href={option.link}
+								key={`wilder-options-${option.link}`}
+								onClick={onClose}
+							>
+								<img src={option.icon} alt={option.label} />
```

Comment:
I also think we should add title attribute to img tags so a user can hover over and see what the icon is for

Reply:
Agreed. Design had a tooltip at first, and I believe it is good to have tooltip in UI/UX